From d633343a1d4d319d97469edc82c6c06929ef6b3d Mon Sep 17 00:00:00 2001
From: Zahari Doychev <zahari.doychev@intel.com>
Date: Thu, 3 Sep 2020 19:45:45 +0200
Subject: [PATCH] PONRTSYS-7917 drivers: cqm change fsqm T3 threshold

The old value 0xa1 was still causing ACM drops and causing e.g. SP in DS to
leak high priority packets.
---
 drivers/net/ethernet/lantiq/cqm/prx300/cqm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c b/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c
index b221eedefce3..13e5f210e3de 100644
--- a/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c
+++ b/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c
@@ -2819,9 +2819,9 @@ static void init_fsqm(void)
 	/* FSQT4 should be 2x(Number of 128B segments for the largest packet)
 	 * for supporting 9.K Jumbo buffer plus some safety margin, we set it to
 	 * 0xa0.FSQT4=FSQT3 will cause back-pressure when the threshold is
-	 * reached. So set FSQT3 to 0xa1
+	 * reached. So set FSQT3 to 0xa3
 	 */
-	cbm_w32((fsqm + FSQT3), 0xa1);
+	cbm_w32((fsqm + FSQT3), 0xa3);
 	cbm_w32((fsqm + FSQT4), 0xa0);
 	cbm_w32(fsqm + IO_BUF_RD, 0);
 	cbm_w32(fsqm + IO_BUF_WR, 0);
